<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainView</class>
 <widget class="QMainWindow" name="mainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>939</width>
    <height>571</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>939</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>快证件照</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="gbox_bgcolor">
       <property name="title">
        <string>底色</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="btn_rbg">
          <property name="text">
           <string>红色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="btn_bbg">
          <property name="text">
           <string>蓝色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="btn_wbg">
          <property name="text">
           <string>白色</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="gb_size">
       <property name="title">
        <string>尺寸</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QRadioButton" name="btn_twoInch">
          <property name="text">
           <string>2寸</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="btn_StwoInch">
          <property name="text">
           <string>小2寸</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="btn_oneInch">
          <property name="text">
           <string>1寸</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="btn_LoneInch">
          <property name="text">
           <string>大1寸</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="btn_SoneInch">
          <property name="text">
           <string>小1寸</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="btn_LtwoInch">
          <property name="text">
           <string>大2寸</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="btn_beauty">
       <property name="text">
        <string>美化</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_exit">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_save">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="wid_hole" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QLabel" name="lab_preview">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-color: rgb(255, 170,0);
     border-width: 1px;
     border-style: solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lab_sizeShow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
